packages/terra-dynamic-grid/tests/nightwatch/dynamic-grid-spec.js

```diff
@@ -2,7 +2,7 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
 
-module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+module.exports = resizeTo(['tiny', 'medium', 'huge'], {
```

Comment:
we probably only need tiny/large here.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1166/commits/f527b4d8133d9be4b6f75715c93a7bf3ec3550be